src/importexport/export/internal/exportconfiguration.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+* Audacity: A Digital Audio Editor
+*/
+
+#include <QDir>
+
+#include "global/settings.h"
+#include "log.h"
+
+#include "types/exporttypes.h"
+
+#include "exportconfiguration.h"
+
+using namespace au::importexport;
+
+static const std::string module_name("export");
+
+static const muse::Settings::Key EXPORT_PROCESS(module_name, "importexport/process");
+static const muse::Settings::Key EXPORT_FILENAME(module_name, "importexport/filename");
+static const muse::Settings::Key EXPORT_DIRECTORY_PATH(module_name, "importexport/directoryPath");
+static const muse::Settings::Key EXPORT_FORMAT(module_name, "importexport/format");
+static const muse::Settings::Key EXPORT_CHANNELS(module_name, "importexport/channels");
+static const muse::Settings::Key EXPORT_SAMPLE_RATE(module_name, "importexport/defaultProjectSampleRate");
+static const muse::Settings::Key EXPORT_ENCODING(module_name, "importexport/encoding");
+
+void ExportConfiguration::init()
+{
+    muse::settings()->setDefaultValue(EXPORT_PROCESS, muse::Val(ExportProcessType::FULL_PROJECT_AUDIO));
+    muse::settings()->valueChanged(EXPORT_PROCESS).onReceive(nullptr, [this] (const muse::Val& val) {
+        m_processChanged.notify();
+    });
+
+    //! NOTE: "untitled" filename is set on every project load
+    muse::settings()->setDefaultValue(EXPORT_FILENAME, muse::Val("untitled"));
```

Comment:
As I see it, "untitled" in AU3 is translatable, so we should translate it as well. It’s better to handle the translation in the model, while the configurator should simply return an empty string.